test/Rx/Functional/Observable/IteratorObservableTest.php

```diff
@@ -104,6 +104,44 @@ public function generator_dispose()
         ], $results->getMessages());
     }
 
+    /**
+     * @test
+     *
+     * @link https://github.com/ReactiveX/RxPHP/issues/39
+     */
+    public function splObjectStorage_many()
+    {
+
+        if (defined('HHVM_VERSION') && version_compare(HHVM_VERSION, '3.11.0', 'lt')) {
```

Comment:
@mbonneau at this point, I don't think it makes any sense to support older versions of hhvm